dashboard/test/controllers/api/v1/regional_partners_controller_test.rb

```diff
@@ -228,4 +228,31 @@ class Api::V1::RegionalPartnersControllerTest < ActionController::TestCase
       assert_equal(25, JSON.parse(@response.body)['capacity'])
     end
   end
+
+  test 'find regional partner summer workshops for specific zip' do
+    regional_partner = create :regional_partner_beverly_hills
+
+    get :find, zip_code: 90210
+    assert_response :success
+    assert_equal regional_partner.contact_name, JSON.parse(@response.body)['contact_name']
+  end
+
+  test 'find regional partner summer workshops for state fallback' do
+    mock_illinois_object = OpenStruct.new(state_code: "IL")
+    Geocoder.stubs(:search).returns([mock_illinois_object])
```

Comment:
Use `expects` instead of `stubs` so we also verify it was actually called.

On that note, add the following code to the above test case to verity the Geocoder is _not_ called:
```ruby
Geocoder.expects(:search).never
```